app/models/nic/interface.rb

```diff
@@ -44,6 +44,22 @@ def vlanid
       return ''
     end
 
+    def mtu
+      # Determine a mtu according to the following cascading rules:
+      # 1. if the interface has a v4 subnet with a non-blank mtu, use that
+      # 2. if the interface has a v6 subnet with a non-blank mtu, use that
+      # 3. if no reasonable mtu was determined, then return nil
+      #
+      # In case the v4 and v6 subnet are both present, they should have the same
+      # mtu. If they have a different mtu, this is probably an error in the
+      # user's database. There is no way of determining the real mtu, so we
+      # pick the v4 one unless it turns out to be blank.
```

Comment:
It would be great to display a warning to the user if they chose two subnets with different mtu for single interface but I don't insist.

Reply:
Yes, and it would be nice to have a warning in case of compute resource network with different MTU too. I can manage the oVirt part (and libvirt if needed), but I don't have any openstack/vmware/aws/azure compute resource to test other implementations.

It would be nice for me if we can keep this warning things out of this PR, to let it as simple as possible :)

Reply:
I didn't mean to ask compute resource about the setting. What I meant is that if you choose in Foreman subnet for ipv4 and ipv6 for a single interface, we could check what MTU is set for them and display warning if these two subnets have different value. It would only be based on Foreman data.

Reply:
Yes, sorry maybe my sentence was not clear. I was trying to say that we can check that both subnet's mtu are identical and in the case we are provisioning an VM, we can add another check with the compute resource network mtu too, as when we configure the nic for a VM, we already get the compute resource network informations.

Reply:
ok, so I suppose the outcome is we'll keep it for later